app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,6 @@
+# Display homepage
+class HomepageController < ApplicationController
+  # GET /
+  def index
+  end
+end
```

Comment:
This is the former `StatsController`, now renamed so its role in the app is clearer.